app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -15,6 +15,10 @@ def provider_collection(manager)
         connection.api.projects
       end
     end
+
+    def to_notify
```

Comment:
Since this returns a boolean, it should probably be a `?` method.  Maybe `notify_on_provider_interaction?`